lua/rose-pine.lua

```diff
@@ -667,13 +667,14 @@ local function set_highlights()
 		TelescopeTitle = { fg = palette.foam, bold = styles.bold },
 
 		-- ibhagwan/fzf-lua
-		FzfLuaNormal = { link = "NormalFloat" },
-		FzfLuaTitle = { fg = palette.foam, bold = styles.bold },
 		FzfLuaBorder = make_border(),
-		FzfLuaHeaderText = { fg = palette.love },
-		FzfLuaHeaderBind = { fg = palette.rose },
-		FzfLuaBufFlagCur = { fg = palette.subtle },
 		FzfLuaBufFlagAlt = { fg = palette.subtle },
+		FzfLuaBufFlagCur = { fg = palette.subtle },
+		FzfLuaCursorLine = { fg = palette.text, bg = palette.overlay },
+		FzfLuaHeaderBind = { fg = palette.rose },
+		FzfLuaHeaderText = { fg = palette.love },
+		FzfLuaNormal = { fg = palette.text, bg = palette.surface },
+		FzfLuaTitle = { fg = palette.foam, bg = palette.surface, bold = styles.bold },
```

Comment:
Let's link these two groups so they respect both `groups.panel` and `styles.transparency`. This also fixes the title having a different background than the borders :)

```suggestion
		FzfLuaNormal = { link = "NormalFloat" },
		FzfLuaTitle = { link = "FloatTitle" },
```

Reply:
Resolved in latest commit.